src/infoview.ts

```diff
@@ -36,26 +38,34 @@ class InfoDocument implements Disposable {
     }
 
     get text() {
-        return this.lines.join('\n');
+        return this.contents;
     }
 
     dispose() {
         for (const s of this.subscriptions) s.dispose();
     }
 
     private render() {
-        this.lines = [];
-        this.renderHeader();
-        this.renderGoal();
-        this.renderMessages();
+        this.contents = this.renderHeader() + this.renderGoal() + this.renderMessages();
+    }
+
+    private stylesheet() {
+        return `#settings { display: block; text-align: right; }
+            h1 {
+                display: block;
+                background-color: blue;
```

Comment:
I think we can get all the styling information from the editor itself, it would be good if we could use a style consistent with the user's preferences.